octopoes/octopoes/connector/octopoes.py

```diff
@@ -167,3 +167,6 @@ def get_finding_type_count(self, valid_time: Optional[datetime] = None) -> Dict[
         params = {"valid_time": valid_time}
         res = self.session.get(f"/{self.client}/finding_types/count", params=params)
         return res.json()
+
+    def recalculate_bits(self) -> int:
+        return self.session.get(f"/{self.client}/bits/recalculate").json()
```

Comment:
This should become a POST as well

Reply:
Perhaps we could add a test for this so we don't miss these?

Reply:
Fixed and added robot test